### PR TITLE
Add back .psci file support

### LINKED ISSUE #2734: Read commands from .psci file when starting session
In the past, PSCi would check for a ".psci" file in the project directory or `~` directory, then execute the commands in their when starting a PSCi session. This was removed several months ago (I can't find the issue/PR), but it would be nice to have that back. One motivation for it is to have `pulp init` create a ".psci" file having "import Prelude". With this, newcomers can do `pulp psci` followed by `1+1` and see a success case, rather than `Unknown operator (+)` error.

### PR DESCRIPTION
**Pull request by paf31**
Fixes #2734

It looks like this:

```
$ echo "import Prelude" > .psci
$ psc-package repl
PSCi, version 0.11.0
Type :? for help

import Prelude
> 1 + 1
2
```

cc @chexxor

### CODE REVIEW

**Comment by hdgarrood** [comment]
Does this allow you to put basically anything inside .psci? Not that I feel particularly strongly that we should be restricting it, but it might be worth discussing.

**Comment by paf31** [comment]
Yes it does right now. I think it might be better to limit it to imports only. There is also a security risk associated with running arbitrary code on behalf of the user.

**Comment by paf31** [comment]
Fixed.

File: app/Command/REPL.hs [hunk 1]
```diff
@@ -347,14 +352,28 @@ command = loop <$> options
...
+                      configFile <- (</> ".psci") <$> liftIO getCurrentDirectory
+                      exists <- liftIO $ doesFileExist configFile
+                      when exists $ do
+                        ls <- lines <$> liftIO (readUTF8File configFile)
+                        for_ ls $ \l -> do
+                          liftIO (putStrLn l)
+                          case parseCommand l of
+                            Left err -> liftIO (putStrLn err >> exitFailure)
+                            Right cmd@Import{} -> handleCommand' state cmd
+                            Right _ -> liftIO (putStrLn "Only import declarations are supported in the .psci file" >> exitFailure)
```
Comment (hdgarrood) [inline app/Command/REPL.hs]: Might be nicer to print a warning, ignore declarations which aren't imports, and continue?

Comment (paf31) [inline app/Command/REPL.hs]: Sounds good

**Comment by hdgarrood** [comment]
:+1:

**Comment by kRITZCREEK** [comment]
`.psci` seems inconsistent after `purs repl` call it `.purs-repl`?

**Comment by paf31** [comment]
Yeah, that makes sense. It is called PSCi elsewhere though.

**Pull request merged**